komodor/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "995fe904-e761-4f2f-8dbf-148baf3f080a",
+  "app_id": "komodor",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Offering::Integration"
+  ],
```

Comment:
Looks like the CI validations are failing here. Can you please try moving the `classifier_tags` key down to be nested inside the `tile` key? 

For reference: https://docs.datadoghq.com/developers/integrations/check_references/?tabs=manifestversion2#manifest-file